File: js/programmers/level.0/prgm_120923_1.js
function solution(num, total) {
  let result = [];
  const averageNum = Math.ceil(total / num);
  const minusNum = Math.trunc(num / 2);
  const startNum = averageNum - minusNum;

  for (i = 0; i < num; i++) {
    result.push(startNum + i);
  }

  return result;
}

File: python/programmers/level.0/prgm_120923_1.py
import math

def solution(num, total):
    result = []
    average_num = math.ceil(total / num)
    minus_num = math.trunc(num / 2)
    start_num = average_num - minus_num

    for i in range(num):
        result.append(start_num + i)
    
    return result